app/screens/Terms.tsx

```diff
@@ -203,93 +208,85 @@ const Terms = ({ navigation }: Props): any => {
+            <View style={styles.vectorImageView}>
+              <VectorImage source={item.image} />
+            </View>
+          </View>
+          <ShiftFromTop15>
+            <Heading2Centerw>{t('walkthroughTextssubtitle0')}</Heading2Centerw>
+          </ShiftFromTop15>
+          <View style={styles.containerView2}>
+            <FeatureTCView
+              title={t('walkthroughTextstitle3').toString()}
+              subTitle={t('walkthroughTextssubtitle3').toString()}
+              iconname={item.advice}
+            />
+            <FeatureTCView
+              title={t('walkthroughTextstitle2').toString()}
+              subTitle={t('walkthroughTextssubtitle2').toString()}
+              iconname={item.tools}
+            />
+            <FeatureTCView
+              title={t('walkthroughTextstitle1').toString()}
+              subTitle={t('walkthroughTextssubtitle1').toString()}
+              iconname={item.activity}
+            />
+          </View>
+          <View>
+            <SideSpacing10>
+              <ButtonTermsRow>
+                <FormOuterTermsCheckbox
+                  onPress={(): any => {
+                    setToggleCheckBox(!toggleCheckBox);
+                  }}>
+                  <CheckboxItem>
+                    <View>
+                      {toggleCheckBox ? (
+                        <CheckboxActive>
+                          <Icon name="ic_tick" size={12} color="#fff" />
+                        </CheckboxActive>
+                      ) : (
+                        <Checkbox></Checkbox>
+                      )}
+                    </View>
+                  </CheckboxItem>
+                  <SideRightSpacing20>
+                    <LabelTextTerms>
+                      {t('tNccheckbox2')}{' '}
+                      <LabelTextTerms onPress={goToPrivacyPolicy} style={styles.privacyText}>
+                        {t('tNcprivacyPolicyTitle')}{' '}
+                      </LabelTextTerms>
+                      {t('childInfoAndText')}{' '}
+                      <LabelTextTerms onPress={goToTerms} style={styles.privacyText}>
+                        {t('tNcheader')}
+                      </LabelTextTerms>
+                      .
                     </LabelTextTerms>
-                    .
-                  </LabelTextTerms>
-                </SideRightSpacing20>
-              </FormOuterTermsCheckbox>
-              <ButtonPrimary
-                disabled={isButtonDisabled}
-                onPress={(): any => {
-                  acceptTerms();
-                }}>
-                <ButtonUpperCaseText numberOfLines={2}>{t('continueCountryLang')}</ButtonUpperCaseText>
-              </ButtonPrimary>
-            </ButtonTermsRow>
-
-          </SideSpacing10>
+                  </SideRightSpacing20>
+                </FormOuterTermsCheckbox>
+                <ButtonPrimary
+                  disabled={isButtonDisabled}
+                  onPress={(): any => {

```

Comment:
Return type is void, you can remove "any" for clarity.

Reply:
okay I will remove this